modules/contracts/adapters/dex/Cargo.toml

```diff
@@ -31,7 +31,7 @@ juno = [
   "dep:cw20_junoswap",
   "abstract-wyndex-adapter/full_integration",
 ]
-terra = ["dep:terraswap", "abstract-astroport-adapter/full_integration"]
+terra2 = ["dep:terraswap", "abstract-astroport-adapter/full_integration"]
 neutron = ["abstract-astroport-adapter/full_integration"]
 osmosis = ["abstract-osmosis-adapter/full_integration"]
```

Comment:
package.metadata.optimizer still has only "terra"

Reply:
Can you handle this in #88 ?

Reply:
Sure

Reply:
nvm, fixed it.